composer.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "ronasit/laravel-media",
+  "description": "A module for working with Media.",
+  "keywords": [
+    "framework",
+    "laravel",
+    "media"
+  ],
+  "license": "MIT",
+  "authors": [
+      {
+          "name": "Ronas IT",
+          "homepage": "https://github.com/RonasIT/laravel-media/graphs/contributors"
+      }
+  ],
+  "require": {
+    "php": "^8.1",
+    "laravel/framework": "^10.0",
+    "php-open-source-saver/jwt-auth": "^2.1",
+    "ronasit/laravel-helpers": "2.5"
```

Comment:
```suggestion
    "ronasit/laravel-helpers": ">=2.5"
```

Reply:
Done.

Reply:
please update code base to work with the minimal version of the Laravel helpers package 2.6 :)

Reply:
I updated the package code to work with the `ronasit/laravel-helpers` package version `2.6`. I think then in composer.json it’s better to write not `>=2.5`, but `>=2.6`. I did it.